bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -962,7 +965,9 @@ public void asyncCreateLedgerAdv(final int ensSize, final int writeQuorumSize, f
                 return;
             }
             new LedgerCreateOp(BookKeeper.this, ensSize, writeQuorumSize,
-                               ackQuorumSize, digestType, passwd, cb, ctx, customMetadata).initiateAdv((long)(-1));
+                               ackQuorumSize, digestType, passwd, cb, ctx,
+                               customMetadata, EnumSet.noneOf(WriteFlag.class))
+                                       .initiateAdv((long)(-1));
```

Comment:
since you are touching this line, can you change `(long)(-1)` to `-1L`?

Reply:
done